README.md

```diff
@@ -1,2 +1,127 @@
+
+Example request `http://localhost:4000/api/mint?walletAddress=rpHJL8bRW81shYm3LwwsXovABps6SM51RS&tokenCount=5&url=ipfs://QmQDDD1cNgnyhPC4pBLZKhVeu12oyfCAJoWr1Qc1QgbkPN&title=test_title&desc=test_description&loc=Warsaw`
+
+⚫ `GET /api/claim` - Checks if user is eligible for NFT claim or creates new offer that allows for claiming NFT for specific event. Endpoint returns response with sellOfferID that has to be accepted by user afterwards.
+
+**@param {string} walletAddress** - The wallet address of the user trying to claim
+
+**@param {integer} type** - The type of claim (1 for checking claim status and metadata, 2 for claiming)
+
+**@param {string} minter** - The minter address of the event
+
+**@param {string} eventId** - The ID of the event
+
+**Possible responses**
+
+    -`404` - Event with specified ID does not exist
+
+    -`claimed` - The NFT for specified event was already claimed by this user
+
+    -`empty` - All NFTs for this particular event were already claimed
+
+    -`success` - If `onlyCheckStatus` parameter was true it indicated that current parameters for selected event were sent
+
+    -`transferred` - Indicates that new sell offer for NFT related to selected event was created successfully and details were sent to user
+
+Example request `http://localhost:4000/api/claim?walletAddress=rpHJL8bRW81shYm3LwwsXovABps6SM51RS&type=2&minter=rMBU1wwAJZZz36YBNxug7YYHKeC5k3vqfG&eventId=0`
+
+⚫ `GET /api/startVerification` - Starts ownership verification process by generating unique ID for user that has to later be included in a Memo of signed tx.
+
+**@param {string} walletAddress** - Wallet address from user requesting verification
+
+Example request `http://localhost:4000/api/startVerification?walletAddress=raY33uxEbZFg7YS1ofFRioeENLsVdCgpC5`
+
+⚫ `GET /api/verifyOwnership` - Verifies whether or not user owns NFT with provided id for particular event from `minter` account.
+
+**@param {string} walletAddress** - Wallet address from user requesting verification
+
+**@param {string} signature** - Signature that should be signed by the same account as walletAddress. This could be done either using XUMM or `sign` function from xrpl library. The mock transaction from signature has to contain memo with number generated for walletAddress. See test.js for example implementation of this
+
+**@param {string} minter** - The minter address of the event
+
+**@param {string} eventId** - The ID of the event
+
+Example request `http://localhost:4000/api/verifyOwnership?walletAddress=raY33uxEbZFg7YS1ofFRioeENLsVdCgpC5&signature=12000022800000002401469CC8201B00779113614000000000C65D4068400000000000000D7321ED57D41105FC480545763677D2100C8949324A97811FE5CB45594B5E73991BBF927440625FEB2E8DEA86736368DE716C0223586C97C44519B45500653F7C505DA9C24F3F8789BBEE917871DC862DB826ACE7B64AB485BADB0C46776E931EA0EC55840381143CBB7B1E0212681492733BDA77986A6A7C4C2B4A83144B4E9C06F24296074F7BC48F92A97916C6DC5EA9&minter=raY33uxEbZFg7YS1ofFRioeENLsVdCgpC5&eventId=0`
```

Comment:
Also, it's quite hard to debug what is incorrect when using `verifyAccount` - is it that my issuer doesn't exist? That my signed transaction isn't correct? Did I forget to include the `startVerification` memo? Did I accidentally provide the wrong number of parameters? (I spent a while trying to get this to work with a transaction I signed locally, but couldn't figure it out before the end of my day unfortunately - I'd recommend adding more clear errors to help people in the future debug where their integration is running into trouble :) )

Will continue testing on Monday :)